wgs_somatic-run-wrapper.py

```diff
@@ -193,13 +196,20 @@ def wrapper():
     # also, if fastqs come from more than one run, what will the value of this argument be then to be "correct"?...
 
 
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--instrument', help='novaseq_687_gc or novaseq_A01736', required=True)
```

Comment:
Instrument shouldn't refer to those specifically. What if we get another machine? Maybe the hint could just be "For example novaseq_687_gc"?

Reply:
solved in commit @ d53179c